### PR TITLE
[INCR-85] Support testing Python 3 in edx-search

### PR DESCRIPTION
**Pull request by cclauss**
https://openedx.atlassian.net/browse/INCR-85

Opened #64

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3115](https://openedx.atlassian.net/browse/OSPR-3115) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@cclauss Thank you for your contribution. Please let me know once it all checks are green and it is ready for review. Thanks

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+      env: TOXENV=quality
+  exclude:
+    - python: 2.7
+      env: TOXENV=django20
+    - python: 2.7
+      env: TOXENV=django21
+  allow_failures:
+    - python: 3.6
+    - python: 3.7
+    - env: TOXENV=quality
```
Comment (jmbowman) [inline .travis.yml]: Hmm, we don't want to leave the quality check failing for Python 2.7.  It looks like the failures are due to new [checks introduced in pylint-django 0.11](https://github.com/PyCQA/pylint-django/blob/master/CHANGELOG.rst#version-011-18-april-2018-the-testcon-moscow-edition); could you take a stab at fixing those?  It looks like it should only require minor changes in `search/views.py`.

Comment (cclauss) [reply .travis.yml]: Fixed.

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-search&entry.867055334=https://github.com/edx/edx-search/pull/63&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-02-22+13:32&entry.1917517419=2019-02-22+18:22&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**